staging/src/k8s.io/client-go/examples/third-party-resources/apis/tpr/v1/register.go

```diff
@@ -24,28 +24,25 @@ import (
 
 var (
 	SchemeBuilder = runtime.NewSchemeBuilder(addKnownTypes)
-	AddToScheme   = SchemeBuilder.AddToScheme
 )
 
-// GroupName is the group name use in this package
+// AddToScheme applies all the functions stored in the SchemeBuilder value to a scheme.
+func AddToScheme(s *runtime.Scheme) error {
```

Comment:
I'll do as you suggest, but I'd also like to discuss why we're exporting `*runtime.SchemeBuilder` instances elsewhere.

Looking at [one example](https://github.com/caesarxuchao/kubernetes/blob/7c1f90e74b1c85929550f5444eb9885e42a04469/pkg/apis/apps/v1beta1/register.go) from #44784, it would also be possible to instead export not a `SchemeBuilder` variable from the package, but rather export a `Register` function.

In package "k8s.io/api/apps/v1beta1":
```go
var schemeBuilder ...

func RegisterWithScheme(funcs ...func(*runtime.Scheme) error) {
  return schemeBuilder.Register(funcs...)
}
```
In [the `init` function of example noted above](https://github.com/caesarxuchao/kubernetes/blob/7c1f90e74b1c85929550f5444eb9885e42a04469/pkg/apis/apps/v1beta1/register.go#L40-L45):
```go
func init() {
	v1beta1.RegisterWithScheme(v1beta1.AddKnownTypes, addDefaultingFuncs, addConversionFuncs)
}
```
With that technique, no caller can rebind the `*runtime.SchemeBuilder`.